libraries/api-extractor-model/src/mixins/ApiItemContainerMixin.ts

```diff
@@ -103,6 +104,7 @@ export function ApiItemContainerMixin<TBaseClass extends IApiItemConstructor>(
     public readonly [_members]: ApiItem[];
     public [_membersSorted]: boolean;
     public [_membersByContainerKey]: Map<string, ApiItem>;
+    public shouldSortMembers: boolean = true;
```

Comment:
Made private, leaving unresolved to bring attention to this.

Reply:
> I'm also not sure if this needs to be a fancy `[_shouldSortMembers]` Symbol property

Yes, it does. The reasons is that `private` members of a base class must not conflict with `private` members of the child class, and with mixins the compiler cannot detect such conflicts.  (In the latest ECMAScript the `#` prefix is another possible solution, however it can't be transpiled efficiently, so `Symbol` is more widely supported across JavaScript runtimes.)

I can fix this.

Reply:
Fixed